### PR TITLE
just: 1.11.0 -> 1.13.0

### PR DESCRIPTION
**Pull request by adamcstephens**
just: 1.11.0 -> 1.13.0
just: add adamcstephens as maintainer

https://github.com/casey/just/blob/1.13.0/CHANGELOG.md

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by xrelkd — APPROVED** [review]

**Comment by xrelkd** [comment]
Result of `nixpkgs-review pr 212606` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>dogdns</li>
    <li>just</li>
    <li>kabeljau</li>
  </ul>
</details>

**Comment by xrelkd** [comment]
Hi @bobby285271, I think this PR is ready to merge. Thank you!!

**Comment by adamcstephens** [comment]
Result of `nixpkgs-review pr 212606` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>dogdns</li>
    <li>just</li>
    <li>kabeljau</li>
  </ul>
</details>

----

Result of `nixpkgs-review pr 212606` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>dogdns</li>
    <li>just</li>
  </ul>
</details>

----

Result of `nixpkgs-review pr 212606` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>dogdns</li>
    <li>just</li>
  </ul>
</details>

File: pkgs/development/tools/just/default.nix [hunk 1]
```diff
@@ -10,21 +10,21 @@
...
   };
 
-  cargoHash = "sha256-pgktCRxYlxo/kMdBm3/dzdU3rOXSNFY9IIE8j8Dgro0=";
+  cargoHash = "sha256-91C/5m2avsW7GKQDg/Ez9fzzFhe8ih1De1RbV/MBJbM=";
 
   nativeBuildInputs = [ installShellFiles ];
   buildInputs = lib.optionals stdenv.isDarwin [ libiconv ];
 
-  nativeCheckInputs = [ coreutils bash ];
+  checkInputs = [ coreutils bash ];
```
Comment (figsoda) [inline pkgs/development/tools/just/default.nix:28]:
```suggestion
```

Comment (06kellyjac) [reply pkgs/development/tools/just/default.nix:28]: The checks certainly require coreutils & IIRC require bash. But this should remain native

Comment (figsoda) [reply pkgs/development/tools/just/default.nix:28]: doesn't both already exist in stdenv.mkDerivation?

Comment (adamcstephens) [reply pkgs/development/tools/just/default.nix]: Yes, the checks run without this. Updated.

**Comment by adamcstephens** [comment]
Rebuilt successfully on `{x86_64,aarch64}-{linux,darwin}`

**Review by xrelkd — APPROVED** [review]
LGTM

**Review by figsoda — APPROVED** [review]

**Pull request merged**